tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -399,7 +399,7 @@ public void getCapacityBytes() {
   @Test
   public void lastModificationTimeCompleteFile() throws Exception {
     long fileId = mFsMaster.createFile(new AlluxioURI("/testFile"), CreateFileOptions.defaults());
-    long opTimeMs = TEST_CURRENT_TIME;
+    long opTimeMs = System.currentTimeMillis() + TEST_TIME_OFFSET_MS;
```

Comment:
I would prefer to have a race on last modification time instead of the added complexity